### ISSUE TITLE
Change player to Mplayer

### ISSUE THREAD

**Issue by Duckduckgot** [Dellai-V/WebRadio#1]
On windows 7 sp1, streams with he aac v2 and some mp3s with a frequency of 48000 cannot be played, but on windows 10 everything plays fine.

**Comment by Duckduckgot** [comment]
@Dellai-V

**Comment by Dellai-V** [comment]
it is definitely a codec problem, try to install: https://www.microsoft.com/en-us/download/details.aspx?id=507

**Comment by Duckduckgot** [comment]
> it is definitely a codec problem, try to install: https://www.microsoft.com/en-us/download/details.aspx?id=507

No, it is for Windows XP, it is not even installed on Windows 7.

**Comment by Dellai-V** [comment]
this program depends on windows media player, you should find the audio codec to integrate with your WMP
